docs/cookbook/usage_tips.rst

```diff
@@ -31,13 +31,15 @@ Multi-threading
 The ShotGrid API is not thread-safe. If you want to do threading we strongly suggest that you use 
 one connection object per thread and not share the connection.
 
+.. _entity-fields:
+
 *************
 Entity Fields
 *************
 
-When you do a :meth:`~shotgun_api3.Shotgun.find` call that returns a field of type entity or 
-multi-entity (for example the 'assets' column on Shot), the entities are returned in a standard 
-dictionary::
+When you do a :meth:`~shotgun_api3.Shotgun.find` or a :meth:`~shotgun_api3.Shotgun.create` call
+that returns a field of type **entity** or **multi-entity** (for example the 'assets' column on Shot),
+the entities are returned in a standard dictionary::
```

Comment:
```suggestion
the entities are returned in a standard dictionary:
```

Reply:
Thanks for the feedback. I think this one is not applicable because next line is an indented code line and if I understand correctly, belongs to the [literal block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) format.

Reply:
Gotcha, thanks for clarifying!